.github/workflows/unit_tests.yaml

```diff
@@ -57,12 +40,11 @@ jobs:
         run: python3 -m unittest unit_test/**/*.py
       - name: Test Seqr Chart Installation
         run: |
-          # This hack lets us skip some expensive setup within the seqr startup
-          export PGPASSWORD='super-secure-password'; psql -h localhost -U postgres -c 'CREATE DATABASE seqrdb'; psql -h localhost -U postgres -c 'CREATE DATABASE reference_data_db';
```

Comment:
I _think_ the correct usage of `startupProbe` added several months ago is rendering this unnecessary.  The standard flow (running all migrations + a few manage.py commands) works in under 2min!